code/modules/clothing/chameleon.dm

```diff
@@ -153,6 +156,8 @@
 	return 1
 
 /datum/action/item_action/chameleon/change/proc/emp_randomise()
+	if(istype(target, /obj/item/weapon/gun/energy/laser/chameleon))
+		return	//Please no crash!
```

Comment:
It just fires once a second. WHY ARE YOU RUINING MY FEATURES @kevinz000, FIX THIS IMMEDIATELY.